examples/detail_info.py

```diff
@@ -0,0 +1,105 @@
+            print("location.lon_without_check(): Longitude (x coordinate) without checking if the location is valid (datatype float): " + str(n.location.lon_without_check()))
+            print("valid(): Check that the location is a valid WGS84 coordinate, i.e. that it is within the usual bounds (datatype bool): " + str(n.location.valid()))
+            print("x: X coordinate (longitude) as a fixed-point integer: " + str(n.location.x))
+            print("y: Y coordinate (latitude)  as a fixed-point integer: " + str(n.location.y))
+            print("------------------------------------------------------")
+
+    def way(self, w):
+        if self._way_showed == False:
+            self._way_showed = True
+            print("This is a WAY with the following properties:")
+            self.print_obj_info(w)
+
+            print("nodes: Ordered list of nodes (datatype osmium.osm.WayNodeList) (read-only): " + str(w.nodes))
+            for node in w.nodes:
+                print("    " + str(node))
+                # TODO: how to access lat/lon of nodes?
+
+            print("ends_have_same_id(): True if the start and end node are exactly the same (synonym for is_closed()): " + str(w.ends_have_same_id()) + "=" + str(w.is_closed()))
+            print("ends_have_same_location(): True if the start and end node of the way are at the same location: " + str(w.ends_have_same_location())) # Throws an exception if the location of one of the nodes is missing.
+            print("------------------------------------------------------")
+
+    def area(self, a):
+        if self._area_showed == False:
+            self._area_showed = True
+            print("This is an AREA with the following properties:")
+            self.print_obj_info(a)
+            print("from_way(): Return true if the area was created from a way, false if it was created from a relation of multipolygon type: " + str(a.from_way()))
+            print("is_multipolygon(): Return true if this area is a true multipolygon, i.e. it consists of multiple outer rings: " + str(a.is_multipolygon()))
+            print("num_rings(): Return a tuple with the number of outer rings and inner rings: " + str(a.num_rings()))
+            print("orig_id(): Compute the original OSM id of this object. Note that this is not necessarily unique because the object might be a way or relation which have an overlapping id space: " + str(a.orig_id()))
+
+            print("outer_rings() / inner_rings(): Return an iterator over all outer rings / inner rings of the multipolygon: " + str(a.outer_rings()))
+            for outer_ring in a.outer_rings():
+                print("    " + str(outer_ring))
+                for inner_ring in a.inner_rings(outer_ring): # TODO: check if this nested structure makes sense
+                    print("        " + str(inner_ring))
```

Comment:
Here you could output len(inner_ring), i.e. the number of nodes in the inner ring, as an example usage.